unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -15,6 +15,7 @@ public class AvatarSlotComponentView : BaseComponentView<AvatarSlotComponentMode
         [Header("Configuration")]
         [SerializeField] internal AvatarSlotComponentModel model;
 
+        [SerializeField] internal NftTypeColorSupportingSO typeColorSupporting;
```

Comment:
in terms of design, its a bit weird for me that the view decides whether it should support color or not, depending on the category of nft.. i think it should be a responsibility of the controller

Reply:
I had the loading of this SO in the controller before, but after a feedback received (https://github.com/decentraland/unity-renderer/pull/5028#discussion_r1178716154) we decided to move it to the view and, at the same time. to avoid having the asset into the `Resources` folder.

For now this `NftTypeColorSupportingSO` is a configuration that we're gonig to keep totally local and I don't see much problem on that the view is managing it directly. But if we need to change the approach in the future, it is going to be easy.